usr/bin/xapp-appimage-thumbnailer

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
Well, `#!/usr/bin/env python3` __is__ working for FreeBSD. While `#!/usr/local/bin/python3` would work _for FreeBSD_, it would not work for most Linux distributions.

Reply:
`#!/usr/bin/env python3` means "use the default python3 on the system, independently of which path it is installed at. It is the responsbility of the system/user to configure where `python3` on the `$PATH` points to.

Reply:
> unfortunately we have frequent problems with users installing other versions of python on their systems, which can generate bug reports that are difficult to diagnose

Yes, I can understand that. Maybe the upstream project (this) could have it generic like I suggested, and Linux distributions could ship it with the distribution-provided path to python3 hardcoded?

Reply:
You misunderstand me. It will find the wrong python, and make the application fail, on both FreeBSD and Linux, under the same conditions for each.

Just because on your machine, specifically, it finds the python in /usr/local/bin, does not mean it will do so for other FreeBSD users.

Reply:
I see, thanks for the clarification.

Reply:
A symlink at OS or package level would do the trick.

Reply:
So would patching it at the package level, preferably using a build system. :)

@mtwebster suggested above that it might be a good idea to port to Meson, which seems pretty reasonable. Just have meson configure the files to use whatever `py_mod.find_installation().path()` points to.

Another benefit would be that the module currently installed to the Debian-specific dist-packages/ directory would be installed correctly on other distros. https://mesonbuild.com/Python-module.html#install_sources will handle this difference for you automatically.

Reply:
My 2 cents, Python applications should not need to be compiled. For example, If I had read that to use these thumbnailers I would have had to use Meson, I would not have put them into [helloSystem](https://hellosystem.github.io/). But since all I had to do was to add some Python files, I did. So please don't require Meson. It's fine as-is.

Reply:
Does this rule of yours also apply to programs that "need" to be Makefile'd? Because Meson or Makefile doesn't make that much of a difference, if you're happier having a Makefile run `sed` on the scripts to switch out their shebang lines?

Reply:
Well, what I like about Python is that since it is an interpreted language, I don't  need any build tools, compilers, and the like.  In any case, I think we can close this, since my "simplification" turned out to be not workable so we should stay with what there is now. Thanks!